com.zsmartsystems.zigbee/src/main/java/com/zsmartsystems/zigbee/app/discovery/ZigBeeNodeServiceDiscoverer.java

```diff
@@ -684,6 +693,14 @@ public void run() {
         }
     }
 
+    public boolean isFinished() {
```

Comment:
Can you please add a line or two of Javadoc to these two methods stating the meaning of these conditions?